src/dmd/root/outbuffer.d

```diff
@@ -271,6 +282,20 @@ struct OutBuffer
         offset += nbytes;
     }
 
+    /**
+     * Allocate space, but leave it uninitialized.
+     * Params:
+     *  nbytes = amount to allocate
+     * Returns:
+     *  slice of the allocated space to be filled in
+     */
+    extern (D) char[] fillVoid(size_t nbytes) pure nothrow
```

Comment:
Your description uses "allocate" everywhere, why call it "fill" then?

Reply:
Because of its similarity to `fillZero()`.

Reply:
I think you're right, allocate is a better name.